index.js

```diff
@@ -30,6 +30,9 @@ const run = async () => {
         apiVersion: 'v1',
         kind: 'Secret',
         metadata: {
+          annotations: {
+            "helm.sh/resource-policy": core.getInput('output_helm_policy'),
+          },
           name: config.output_secret_name,
           ...(outputSecretNamespace ? { namespace: outputSecretNamespace } : {}),
         },
```

Comment:
`ncc build index.js --license licenses.txt`

Run this command after complete the changes, since the actions needs to be compiled to properly work

Reply:
More info: 

in the vercel section 
[ncc build index.js --license licenses.txt](https://docs.github.com/es/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github)